client/src/Api/Handlers.elm

```diff
@@ -0,0 +1,98 @@
+module Api.Handlers exposing (..)
+
+import Components.Svg exposing (..)
+import Data.Api as Api
+import Data.PageData as PageData
```

Comment:
> Also, something strange happens to me when such a product is out of stock.
> The "cart counter" is suddenly 0 instead of 1

> And it doesn't seem to be possible to reduce the quantity of the product in the cart without going to the cart in this case

@rvem Fixed this problem, the "AddToCart" now shows correct amount even when adding resulting in out of stock

[Screencast from 08-26-2025 03:37:52 PM.webm](https://github.com/user-attachments/assets/43814977-cd28-460b-b273-706d5a7e5965)